Add CamAIML conference + 1 video of the talk

bzz opened the pull request:
Fixes #62

bzz committed:
Add Talks->Video section

Fixes #62

Signed-off-by: Alexander Bezzubov <bzz@apache.org>

EgorBu commented on README.md:
```diff
@@ -260,6 +261,13 @@ for Modeling Source Code?](http://web.cs.ucdavis.edu/~devanbu/isDLgood.pdf) - Vi
+### Video
+
+* ["Neural Abstract Machines & Program Induction"]( https://www.youtube.com/playlist?list=PLzTDea_cM27LVPSTdK9RypSyqBHZWPywt)  Workshop at NIPS 2016
+* ["Learning to Code: Machine Learning for Program Induction"](https://www.youtube.com/watch?v=vzDuVhFMB9Q) CamAIML 2017 presentation by MSR.
```
Could you modify updates to make style consistent?
As an example:
`Differentiable Neural Computer (DNC) - TensorFlow implementation of the Differentiable Neural Computer.`

EgorBu replied:
Maybe it's worth to add `CamAIML` conference + talks because they have one more related talk
```
Learning to Code: Machine Learning for Program Induction | Alex Gaunt 
Learning Program Representations: Symbols to Vectors to Semantics | Miltos Allamanis
```

What do you think? In this case, we can preserve structure as it is for now.

EgorBu commented on README.md:
```diff
@@ -260,6 +261,13 @@ for Modeling Source Code?](http://web.cs.ucdavis.edu/~devanbu/isDLgood.pdf) - Vi
+
+* ["Neural Abstract Machines & Program Induction"]( https://www.youtube.com/playlist?list=PLzTDea_cM27LVPSTdK9RypSyqBHZWPywt)  Workshop at NIPS 2016
+* ["Learning to Code: Machine Learning for Program Induction"](https://www.youtube.com/watch?v=vzDuVhFMB9Q) CamAIML 2017 presentation by MSR.
+* ["source{d} tech talks - Machine Learning on Source Code"](https://www.youtube.com/playlist?list=PL5Ld68ole7j3iQFUSB3fR9122dHCUWXsy) source{d} tech talks 2017 @Moscow
```
It's already added at ```source{d} tech talks
Talks```, so not needed

EgorBu commented on README.md:
```diff
@@ -260,6 +261,13 @@ for Modeling Source Code?](http://web.cs.ucdavis.edu/~devanbu/isDLgood.pdf) - Vi
 
+### Video
+
+* ["Neural Abstract Machines & Program Induction"]( https://www.youtube.com/playlist?list=PLzTDea_cM27LVPSTdK9RypSyqBHZWPywt)  Workshop at NIPS 2016
```
This link is already presented at ```NIPS Neural Abstract Machines and Program Induction workshop
Talks```. So it's not needed.

EgorBu reviewed (changes requested):
Thank you for a good idea and nice material! Please check the feedback + missed signoff. As it was mentioned in comment - maybe it's better to add new conference with talks instead of new `Video` chapter.

EgorBu reviewed (changes requested):
it should be changed according to comments above

eiso commented:
@bzz minor ping, no rush, just a friendly reminder.

bzz committed:
Address review feedback

 - remove video section
 - create CamAIML in conferences and move only new vide link there

Signed-off-by: Alexander Bezzubov <bzz@apache.org>

bzz commented:
@EgorBu feedback addressed, ready for another pass

EgorBu commented:
LGTM
